secrets-provider-test-chart/values.yaml

```diff
@@ -0,0 +1,25 @@
+global:
+  namespace: local-secrets-provider
+
+# K8s permission defaults
+rbac:
+  roleName: secrets-provider-role
+  roleBinding: secrets-provider-role-binding
+  serviceAccount:
+    # Specifies whether a service account should be created
+    create: true
+    # The name of the service account to use.
+    # If not set and create is true, a name is generated using the fullname template
+    name: cucumber
+
+secretsProvider:
+  image: cyberark/secrets-provider-for-k8s
+  tag: 1.0.0
+  name: cyberark-secrets-provider
+
+secretsProviderEnvironment:
+  conjurSslCertificateValue: ""
```

Comment:
Is this a valid value?

Reply:
yes - it is

Reply:
I will check this one to see if I can move it custom-values